demos/browser-stt/src/main.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import "./style.css";
+import {
+  Playthrough,
+  BrowserSttService,
+  AudioOutputsService,
+  createPlaythroughToken,
+  createConversation,
+  Conversation,
+  MessageCharacter,
+} from "@charisma-ai/sdk";
+
+declare global {
+  interface Window {
+    start: () => Promise<void>;
+    reply: () => void;
+    onKeyPress: (event: KeyboardEvent) => void;
+    toggleMuteBackgroundAudio: () => void;
+    toggleMicrophone: (event: Event) => void;
+  }
+}
+
+const STORY_ID = 17221;
+
+const speaker = new AudioOutputsService(); // For character speech
+const backgroundAudio = new Audio(); // For background audio
```

Comment:
how does this work if multiple background tracks are received?

Reply:
This is something that needs to be looked at and fleshed out in the audio manager. I've made a ticket.